.github/workflows/docs.yml

```diff
@@ -9,11 +9,6 @@ on:
       - '[0-9]+.[0-9]+.[0-9]+'
     branches: 
       - master
-  pull_request:
-      branches:
-      - main
-  release:
```

Comment:
Removed the trigger on release, because it's practically the same as trigger on tag push above. So it was just confusing whether both need to be handled separately, or whether the "tag push" event is triggered also when one makes a release (it does).

I also removed the pull_request trigger. If I remember correctly, this was used for testing the documentation workflow. But for one, `django-components` uses `master`, not `main`, and two, for debugging workflows, people can fork this repo and add this trigger themselves.

Reply:
Removing the pull_request trigger would remove the desired test-build of the docs for PRs or not?

Removing the trigger on release is fine. I also saw that recently.  You can then also remove the final section:

https://github.com/django-components/django-components/blob/dcd4203eeadafc5d500b4b72c5cf04f1fe7317e7/.github/workflows/docs.yml#L76-L80

Reply:
@dalito Thanks for reading through this!

> Removing the pull_request trigger would remove the desired test-build of the docs for PRs or not?

Nope, for testing docs there's [the "test_docs" step here](https://github.com/JuroOravec/django-components/blob/9bdd6155a3ea655edd23b632f3ac2714426dd62a/.github/workflows/tests.yml#L47) . And that still ran with other tests [here](https://github.com/JuroOravec/django-components/actions/runs/13471243317/job/37645017599)